lesson_06/conditionals/conditionals_app/src/main/java/com/codedifferently/lesson6/Lesson6.java

```diff
@@ -21,7 +21,12 @@ public static void main(String[] args) {
    * @return True if the age corresponds to a voting age and false otherwise.
    */
   public static boolean canVote(int age) {
-    return false;
+    if (age >= 18) {
```

Comment:
Would be cleaner as a tenary expression.

Reply:
Misspoke, you don't even need a ternary. You can make this a one-line return statement though.